docs/build-with-0g/compute-network/sdk.md

```diff
@@ -7,179 +7,142 @@ sidebar_position: 3
+The broker is your main interface to the 0G Compute Network. It handles authentication, billing, and service communication.
 
+Before initializing the broker, you need:
+1. A wallet signer (implements either `JsonRpcSigner` or `Wallet` from ethers package)
+2. (Optional) The contract address for the 0G Serving contract
+
+**Option 1: Using a private key**
 ```typescript
+const { ethers } = require("ethers");
+const { createZGServingNetworkBroker } = require("@0glabs/0g-serving-broker");
+
+const provider = new ethers.JsonRpcProvider("https://evmrpc-testnet.0g.ai");
+
+// Get the signer
+const privateKey = "INPUT_PRIVATE_KEY_HERE";
+const wallet = new ethers.Wallet(privateKey, provider);
+
+// Initialize broker
+const broker = await createZGServingNetworkBroker(wallet);
+```
+
+**Option 2: Using a browser wallet**
+```typescript
+import { BrowserProvider } from "ethers";
 import { createZGServingNetworkBroker } from "@0glabs/0g-serving-broker";
 
-/**
- * 'createZGServingNetworkBroker' is used to initialize ZGServingUserBroker
- *
- * @param {JsonRpcSigner | Wallet} signer - A signer that implements the 'JsonRpcSigner' or
- * 'Wallet' interface from the ethers package.
- *
- * @param {string} contractAddress - 0G Serving contract address, use default address if not provided.
- *
- * @returns broker instance.
- *
- * @throws An error if the broker cannot be initialized.
- */
-const broker = await createZGServingNetworkBroker(signer);
+async function initializeBrokerWithWallet() {
+  // Ensure wallet is installed
+  if (typeof window.ethereum !== 'undefined') {
+    // Create a provider
+    const provider = new BrowserProvider(window.ethereum);
+    
+    // Get the signer
+    const signer = await provider.getSigner();
+    
+    // Initialize broker
+    const broker = await createZGServingNetworkBroker(signer);
+  }
+}
 ```
 
-### Step 3: List Available Services
+> 💡 The `contractAddress` parameter is optional - the SDK uses a default address if not provided.
+
+### Discover Available Services
+
+The 0G Compute Network hosts multiple AI service providers. The service discovery process helps you find and select the appropriate services for your needs.
 
 ```typescript
-/**
- * 'listService' retrieves a list of services from the contract.
- *
- * @returns {Promise<ServiceStructOutput[]>} A promise that resolves to an array of ServiceStructOutput
- * objects.
- *
- * type ServiceStructOutput = {
- *   provider: string;  // Provider's wallet address, which is the unique identifier for the provider.
- *   name: string;
- *   serviceType: string;
- *   url: string;
- *   inputPrice: bigint;
- *   outputPrice: bigint;
- *   updatedAt: bigint;
- *   model: string;
- * };
- *
- * @throws An error if the service list cannot be retrieved.
- */
 const services = await broker.listService();
 ```
 
-### Step 4: Manage Accounts
+Each service contains the following information:
+```typescript
+type ServiceStructOutput = {
```

Comment:
```
type ServiceStructOutput = {
  provider: string;  // Provider's wallet address (unique identifier)
  name: string;      // Service name
  serviceType: string; // Type of service
  url: string;       // Service URL
  inputPrice: bigint; // Price for input processing
  outputPrice: bigint; // Price for output generation
  updatedAt: bigint;  // Last update timestamp
  model: string;      // Model identifier
  verifiability: string // Indicates how the service's outputs can be verified. 'TeeML' means it runs with verification in a Trusted Execution Environment. An empty value means no verification.
};
```
Sorry, I missed the 'verifiability' attribute in the original version

Reply:
Thanks @Ravenyjh, pushed the change